etb_format.c

```diff
@@ -52,12 +52,14 @@ static int init_stream(struct stream *stream, struct stream *parent)
     return 0;
 }
 
-int decode_etb_stream(struct stream *etb_stream)
+int decode_etb_stream(struct stream *etb_stream, int unaligned)
 {
     struct stream *stream;
     int ret = 0;
-    int nr_stream, pkt_idx, byte_idx, id, cur_id, pre_id, nr_new, i, trace_stop = 0;
+    int nr_stream, pkt_idx, byte_idx, id, cur_id, pre_id, nr_new, i, b, trace_stop = 0;
+    int ofs = 0;
     unsigned char c, end, tmp;
+    const unsigned char fsync[] = {0xff,0xff,0xff, 0x7f};
```

Comment:
Please add a space after each comma.
fsync[]  = { 0xff, 0xff, 0xff, 0x7f };

Reply:
Fixed in 31443f0.